READMEs/datatokens-flow.md

```diff
@@ -123,4 +123,60 @@ print(f"datatoken name: {erc20_token.token_name()}")
 print(f"datatoken symbol: {erc20_token.symbol()}")
 ```
 
+## 3. Publish datatokens using one helper
```

Comment:
No point in adding a different section and the entire flow all over again. Just say something like "as an alternative to publishing the nft and the datatoken using separate transactions, you can use the ... function to deploy them within a single transaction ... ocean.py also offers the option of creating the NFT, the ERC20 and a pool, exchange or dispenser within the same transaction, using the PoolData, FixedData or DispenserData as arguments".

Then just show samples of the data involved. The user is not stupid, we don't need to copy the whole flow from start to end just to showcase a functionality almost identical to what we already showed them.